e2e/suite.go

```diff
@@ -92,6 +93,7 @@ var e2eGroups = map[string]testhelper.Group{
 	"SECURITY":   security.E2ETests,
 	"SIGN":       sign.E2ETests,
 	"VERIFY":     verify.E2ETests,
+	"OCSPVERIFY": verifyocsp.E2ETests,
```

Comment:
Because the new functionality is entirely part of the `verify` command, I'd prefer these tests are added to the `VERIFY` group / package, rather than a new group created.

We usually want to keep things for a command together, except when there are specific reasons to separate them, or they test functionality that is implemented across multiple commands.